src/Arcus.Security.Core/CompositeSecretProvider.cs

```diff
@@ -225,14 +232,14 @@ private void EnsureAnySecretProvidersConfigured(string secretName)
             SecretStoreSource source, 
             Func<SecretStoreSource, Task<T>> callRegisteredProvider) where T : class
         {
-            /* TODO: use 'Arcus.Observability.Telemetry.Core' 'LogSecurityEvent' instead once the SQL dependency is moved
-                        -> https://github.com/arcus-azure/arcus.observability/issues/131 */
-            _logger.LogInformation("Events {EventName} (Context: {@EventContext})", "Get Secret", new Dictionary<string, object>
+            if (_auditingOptions.EmitSecurityEvents)
             {
-                ["EventType"] = "Security",
-                ["SecretName"] = secretName,
-                ["SecretProviderType"] = source.SecretProvider.GetType().Name
-            });
+                _logger.LogSecurityEvent("Get Secret", new Dictionary<string, object>
+                {
+                    ["SecretName"] = secretName,
+                    ["SecretProviderType"] = source.SecretProvider.GetType().Name
```

Comment:
Maybe just leave out the type as it will give the name or not? Would prefer a name over a type as a user
```suggestion
                    ["SecretProvider"] = source.SecretProvider.GetType().Name
```

Reply:
We could maybe come up with an additional solution as wrapped instances (like cached) wil not show the inner type.